### ISSUE TITLE
Its not able to use python in the backend and next.js in the frontend

### ISSUE THREAD

**Issue by Theaccountantguy** [stackblitz/bolt.new#261]
Hi Developers.

Great product to use. I was looking for a project using python as the backend and next.js with react framework as the frontend as a non-techie.

I tried running it with the help of bolt, but it failed to build the same saying it can't use python frameworks in the web environment.

Can you look into the same and allow people to use both python and next.js as backend and frontend respectively.

Thanks!

**Comment by JKing-8** [comment]
I agree with your point of view. I am more familiar with Python and hope to support Django backend. Now it only supports node-related backends.
